contents/tutorials/churn-rate.md

```diff
@@ -0,0 +1,87 @@
+---
+title: How to calculate and lower churn rate with PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-09-28
+featuredImage: ../images/tutorials/banners/actions.png
+topics: ['trends']
+---
+
+- **Level:** Easy 🦔
+- **Estimated reading time:** 6 minutes ☕️
+
+The word churn once reminded people of fresh butter, now it brings thoughts of lost users and revenue to product and marketing teams.
+
+The churn rate is the percentage of users who stop an action during a certain period. It is used to understand the negative changes in usage, revenue, or other growth metrics. Understanding churn rate is key to figuring out problem areas in your product. 
+
+In this tutorial, we’ll calculate and visualize the churn rate then use PostHog’s features of session recordings, cohorts, and actions to lower it.
+
+### Prerequisites
+
+To follow this tutorial along, you need to have:
+
+1. **[Deployed PostHog](https://posthog.com/docs/deployment)**.
+
+## Calculating churn rate
+
+To calculate churn rate, first, you must define what ended or stopped. For subscription-based businesses, this could be monthly recurring revenue attributable to a subscriber. For a customer success or sales team, it could be organizations usage. For a product team, it might be product usage events by a user. For this tutorial, this product usage is what we’ll focus on.
```

Comment:
```suggestion
To calculate churn rate, first, you must define what ended or stopped. For subscription-based businesses, this could be monthly recurring revenue attributable to a subscriber. For a customer success or sales team, it could be organization usage. For a product team, it might be product usage events by a user. We'll focus on product usage for this tutorial.
```